<launch>
  <include file="$(find wrist_ft_sensor_cleaner)/launch/gravity.launch">
  	<arg name="start_link" value="base_link" />
  	<arg name="end_link" value="_kms40_frame_out" />
  	<arg name="gravity_vector" value="[0,0,-9.8]"/>
  	<arg name="urdf" value="/$(find ur_description)/urdf/ur5_with_gripper.urdf" />
  </include>

  <node name="ft_cleaner" pkg="wrist_ft_sensor_cleaner" type="ft_cleaner.py" output="screen">
  	<param name="ft_topic" value="/kms40/wrench" />
   	<param name="acceleration_topic" value="acceleration" />
  	<param name="path_to_saved_params" value="$(find wrist_ft_sensor_cleaner)/launch/params.yaml" />
  </node>	
</launch>
